pkg/cmd/roachtest/cluster.go

```diff
@@ -3141,3 +3142,66 @@ func (c *clusterImpl) GetHostErrorVMs(ctx context.Context, l *logger.Logger) ([]
 	}
 	return allHostErrorVMs, nil
 }
+
+var sideEyeToken, _ = os.LookupEnv("SIDE_EYE_API_TOKEN")
```

Comment:
nit: there's `os.GetEnv()` if you don't care about the second return value.

Reply:
Yeah, I though we linted against using that to push folks to envutil which mandates the `COCKROACH_` prefix